### DISCUSSION TITLE
Crazyflie & Bolt papers ICRA 2022

### DISCUSSION THREAD

**Discussion by knmcguire** [bitcraze/discussions#53] [Show and tell]
Unfortunately I was not able to join ICRA 2022 but I followed it virtually! 

I found about 10 papers that use either a ⚡ Bolt or 🐝  Crazyflie(s), here separated by sessions, but let me know if I missed any. 

### Aerial Robotics sessions
- ⚡ 'Passive Wall Tracking for a Rotorcraft with Tilted and Ducted Propellers using Proximity Effects' Ding et al. from City University of Hong Kong & Massachusetts Institute of Technology
- 🐝 'A Fast and Efficient Attitude Control Algorithm of a Tilt-Rotor Aerial Platform Using Inputs Redundancies' Su et al. from UCLA
- ⚡'Cooperative Modular Single Actuator Monocopters Capable of Controlled Passive Separation', Cai et al. from Singapore University of Technology & Design
- 🐝'Optimal Inverted Landing in a Small Aerial Robot with Varied Approach Velocities and Landing Gear Designs' Habas et al. from Penn State 
- 🐝 'PogoDrone: Design, Model, and Control of a Jumping Quadrotor', Zhu et al from Lehigh U. ([arxiv](https://arxiv.org/pdf/2204.00207.pdf) )
- 🐝 'Clustering and Informative Path Planning for 3D Gas Distribution Mapping: Algorithms and Performance Evaluation', Ercolani et al from EPFL (RAL)
- 🐝 'A Bimodal Rolling-Flying Robot for Micro Level Inspection of Flat and Inclined Surfaces'  , Pimentel et al from Instituto Superior Tecnico (RAL)

### Collision Avoidance
- 🐝x 2 'Collision Avoidance for Multiple Quadrotors Using Elastic Safety Clearance Based Model Predictive Control', Jin et al.  from USTC & Sina 
- 🐝x 6 'Nearest-Neighbor-Based Collision Avoidance for Quadrotors Via Reinforcement Learning', Ourari et al. from TU Darmstadt **(updated)**

### Manufacturing and Actuation
- 🐝 + ⚡🦋 'An Experimental Study of Wind Resistance and Power Consumption in MAVs with a Low-Speed Multi-Fan Wind System', Olejnik et al. from TU Delft
 
### Multi-Robot and Swarm Robotics
- 🐝x 6 'Formation-containment tracking and scaling for multiple quadcopters with an application to choke-point navigation', Su et al from The University of Manchester

### Planning and Learning
- 🐝x 6 'Safe Multi-Agent Motion Planning Via Filtered Reinforcement Learning',  Vinod et al. from Mitsubishi Electric Research Laboratories   **(updated)**

### Autonomous Vehicle Navigation
- 🐝 'Event-Triggered Tracking Control Scheme for Quadrotors with External Disturbances: Theory and Validations', Goa et al. from University of Shanghai for Science and Technology  **(updated)**

### Learning from Demonstration
 - 🐝 'Watch and Learn: Learning to control feedback linearizable systems from expert demonstrations', Sultangazin et al. from University of California  **(updated)**

### Motion Control
- 🐝'KoopNet: Joint Learning of Koopman Bilinear Models and Function Dictionaries with Application to Quadrotor Trajectory Tracking', Folkestad et al. from Caltech **(updated)**

**Comment by magicbycalvin** [comment]
Just a heads up, on my browser (Brave Browser V1.39.111), the icon you are using to represent the Crazyflie is showing as an empty box (the bolt symbol is working just fine though).

**Reply by knmcguire** [reply]
Ahhh oke! On android the 'flie' looks pretty different then on chrome. How about a bee? 🐝

**Reply by magicbycalvin** [reply]
Yep, the bee works great!

**Reply by knmcguire** [reply]
Done! Thanks for the headsup! 😁 I guess that a butterfly  🦋 was maybe more logical in name, but actually I'll use that for the Bolt based flapper !

**Comment by knmcguire** [comment]
Now that the IEEE xplore pages are out for ICRA papers.. it seems that I missed 5 additional ones! There must be a better way to search through these papers 😅
